tests/components/homematicip_cloud/conftest.py

```diff
@@ -0,0 +1,82 @@
+"""Initializer helpers for HomematicIP fake server."""
+from unittest.mock import MagicMock, patch
+
+from homematicip.aio.connection import AsyncConnection
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.homematicip_cloud import (
+    DOMAIN as HMIPC_DOMAIN,
+    const as hmipc,
+    hap as hmip_hap,
+)
+from homeassistant.core import HomeAssistant
+
+from .helper import AUTH_TOKEN, HAPID, HomeTemplate
+
+from tests.common import MockConfigEntry, mock_coro
+
+
+@pytest.fixture(name="mock_connection")
+def mock_connection_fixture():
```

Comment:
It's possible to drop the `name` parameter to fixture and just call this method `mock_connection`

Reply:
Done

Reply:
We want a different name to avoid a pylint warning and not have to disable that. Clean test runs decreases risk of missing actual problems. We don't want tests to cry wolf every time.

Reply:
Yes, that was the reason why i used the names.